src/languageserverinstance.jl

```diff
@@ -94,11 +97,25 @@ end
 Run the language `server`.
 """
 function Base.run(server::LanguageServerInstance)
+    @async for msg in server.out_msg_queue
+        try
+            write_transport_layer(server.pipe_out, msg, server.debug_mode)
+        catch err
+            Base.display_error(stderr, err, catch_backtrace())
+            rethrow(err)
+        end
+    end
+
     trigger_symbolstore_reload(server)
     
     global T
     while true
         message = read_transport_layer(server.pipe_in, server.debug_mode)
+        
```

Comment:
Do we need to do something more informative here?

Reply:
That's a really unhelp view of the code, you'll have to click through to the file

Reply:
This should only trigger if the input pipe gets closed, and then we just exit the while loop and shutdown. I think whatever we do here doesn't matter much :) I also plan to rewrite the whole read part of our message handling story in a similar way to the write part, so this code won't have a long lifetime.